operator/v1/types_dns.go

```diff
@@ -287,6 +287,17 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+	// protocolPolicy configures the protocol policy for upstream DNS requests.
+	//
+	// This field allows a cluster admin to optionally configure:
+	// * None configures CoreDNS for to use its default upstream protocol selection.
+	// * ForceTCP configures CoreDNS to use TCP for all upstream DNS requests, even if the client request uses UDP.
+	//
+	// Note that this does not affect the protocol of DNS requests made by clients internal to the cluster.
```

Comment:
This is a little confusing to me, and it might be very confusing to someone who isn't already familiar with all the context.  Rather than "made by clients internal to the cluster", how about "between clients and CoreDNS"?
```suggestion
	// Note that this only affects the protocol of DNS requests that CoreDNS makes
	// to upstream resolvers.  It does not affect the protocol of DNS requests between
	// clients and CoreDNS.
```
Same applies to `UpstreamResolvers`.